GarupaPico/GarupaPico/MainPage.xaml

```diff
@@ -1,38 +1,53 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
-<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+<TabbedPage xmlns="http://xamarin.com/schemas/2014/forms"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              xmlns:local="clr-namespace:GarupaPico"
-             x:Class="GarupaPico.MainPage">
+             x:Class="GarupaPico.MainPage" >
+    <!--NavigationPage Title="Cal Tool">
+        <x:Arguments>
+            <local:TabbedCalPage />
+        </x:Arguments>
+    </NavigationPage-->
 
-    <StackLayout>
+    <ContentPage>
+        <StackLayout Padding="20,20">
 
-        <!-- Place new controls here -->
-        <!-- Title -->
-        <Label Text="バンドリ！ガールズバンドパーティ！"
-               HorizontalOptions="Center" />
-        <Label Text="モバイルステーシヨン"
-               HorizontalOptions="Center"
-               VerticalOptions="StartAndExpand" />
+            <!-- Place new controls here -->
+            <!-- Title -->
+            <Label Text="バンドリ！ガールズバンドパーティ！"
+                   HorizontalOptions="Center"
+                   Style="{DynamicResource TitleStyle}"
```

Comment:
Where is this style???

Reply:
https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/text/styles
Referred from here.